### DISCUSSION TITLE
date_format not working.

### DISCUSSION THREAD

**Discussion by RonnieBlaze** [UltimaHoarder/UltimaScraper#821] [General]
whats the purpose of setting the date_format to what i want if when i call the {date} in the folder name it does not use the format i set?

**Comment by UltimaHoarder** [comment]
> whats the purpose of setting the date_format to what i want if when i call the {date} in the folder name it does not use the format i set?

Are you using the latest commit?

**Reply by RonnieBlaze** [reply]
probably not, was waiting till the other issues with the latest comment was fixed.. i will try the current code now

**Reply by UltimaHoarder** [reply]
Use this latest commit, there was an issue where filenames weren't updating

https://github.com/DIGITALCRIMINAL/OnlyFans/commit/5bafbd418b67f10829642262f0c8228d12f58479

**Reply by RonnieBlaze** [reply]
running it now.. will it rename the old folders or are they suck with the old format name?

**Reply by UltimaHoarder** [reply]
It will automatically convert the files as long as it's in the metadata.

**Reply by RonnieBlaze** [reply]
ok cool, ill check it the folders and names when its done running.

**Reply by RonnieBlaze** [reply]
ok it looks like the folder names are not using the "date_format": "%d-%m-%Y" but are using %m-%d-%Y format.

**Reply by UltimaHoarder** [reply]
The %m-%d-%Y format is the one you wanted right?

**Reply by RonnieBlaze** [reply]
its working... i had the wrong config file open..  had the previous release config open.
